test/smoke/src/areas/positron/welcome/welcome.test.ts

```diff
@@ -0,0 +1,210 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2024 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+import { expect } from '@playwright/test';
+import { Application, Logger, PositronPythonFixtures, PositronRFixtures } from '../../../../../automation';
+import { installAllHandlers } from '../../../utils';
+
+/*
+ *  Welcome test cases.
+ */
+
+export function setup(logger: Logger) {
+	describe('Welcome Page', () => {
+		installAllHandlers(logger);
+
+		beforeEach(async function () {
+			const app = this.app as Application;
+
+			await app.workbench.quickaccess.runCommand('Help: Welcome');
+			app.workbench.editors.waitForActiveEditor('Welcome');
+		});
+
+		afterEach(async function () {
+			const app = this.app as Application;
+			await app.workbench.quickaccess.runCommand('View: Close All Editors');
+		});
+
+		it('Verify Welcome page header and footer', async function () {
+			const app = this.app as Application;
+			const logoLocator = '.product-logo';
+			const titleLocator = '.gettingStartedCategoriesContainer div.header div .positron';
+			const footerLocator = '.gettingStartedCategoriesContainer div.footer';
+
+			const footer = app.code.driver.getLocator(footerLocator);
+
+			const logo = app.code.driver.getLocator(logoLocator);
+
+			await expect(logo).toBeVisible();
+			const logoBox = await (await logo.elementHandle())?.boundingBox();
+
+			expect(logoBox?.height).toBeGreaterThan(0);
+			expect(logoBox?.width).toBeGreaterThan(0);
+
+			const title = app.code.driver.getLocator(titleLocator);
+
+			// product name in release is 'Positron' and in dev is 'Positron Dev'
+			await expect(title).toHaveText([/(Positron)|(Positron Dev)/, 'an IDE for data science']);
+
+			await expect(footer).toHaveText('Show welcome page on startup');
+		});
+
+		it('Verify Welcome page content', async function () {
+			const app = this.app as Application;
+
+			const startTitle = app.workbench.positronWelcome.startSection.getByRole('heading');
+			const startButtons = app.workbench.positronWelcome.startSection.getByRole('button');
+
+			const helpSection = app.code.driver.getLocator('.positron-welcome-page-help');
+			const helpTitle = helpSection.getByRole('heading');
+			const helpLinks = helpSection.getByRole('link');
+
+			const openSection = app.code.driver.getLocator('.categories-column.categories-column-right .index-list.start-container');
+			const openTitle = openSection.getByRole('heading');
+			const openButtons = openSection.getByRole('button');
+
+			const recentSection = app.code.driver.getLocator('.categories-column.categories-column-right .index-list.recently-opened');
+			const recentTitle = recentSection.getByRole('heading');
```

Comment:
I would prefer to have these as part of `positronWelcome.ts` so they can be re-used in other tests and then only 1 place to update. Alos keeps this test cleaner.